src/jit/unwindarm.cpp

```diff
@@ -201,11 +401,40 @@ void Compiler::unwindPushMaskFloat(regMaskTP maskFloat)
 {
     // Only floating point registers should be in maskFloat
     assert((maskFloat & RBM_ALLFLOAT) == maskFloat);
+
+#if defined(_TARGET_UNIX_)
+    if (generateCFIUnwindCodes())
+    {
+        regMaskTP regBit = 1;
+
+        for (regNumber regNum = REG_FIRST; regNum < REG_COUNT; regNum = REG_NEXT(regNum), regBit <<= 1)
+        {
+            if (regBit > maskFloat)
+            {
+                break;
+            }
+
+            if (regBit & maskFloat)
+            {
+                unwindPushCFI(regNum);
+            }
+        }
```

Comment:
This loop looks the same as the one for unwindPushMaskInt(). How about creating a function (unwindPushCFIMask()?)

Note that you could optimize the float case by starting at the first float register; currently it's iterating through all the integer registers just to get to the first float reg.

Reply:
(You could also use the BitScanForward/BitScanForward64 intrinsics, if you're really worried about perf)

Reply:
@BruceForstall Donw